eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -25,12 +25,13 @@ public static function dumpAssets(CommandEvent $event)
         $options = self::getOptions($event);
         $appDir = $options['symfony-app-dir'];
         $webDir = $options['symfony-web-dir'];
-        $env = isset($options['ezpublish-asset-dump-env']) ? $options['ezpublish-asset-dump-env'] : '';
+        // @deprecated: Use of ezpublish-asset-dump-env is deprecated, use standard SYMFONY_ENV instead.
```

Comment:
I'm okay with that too, as long as it gets rid of the question. So, using `dev` by default, unless `SYMFONY_ENV` is set.